vpp-manager/startup/interface_config.go

```diff
@@ -113,6 +113,10 @@ func loadInterfaceConfigFromLinux(ifSpec config.InterfaceSpec) (*config.LinuxInt
 	conf.NumTxQueues = link.Attrs().NumTxQueues
 	conf.NumRxQueues = link.Attrs().NumRxQueues
 	conf.Mtu = link.Attrs().MTU
+	if ifSpec.IsMain {
+		config.Info.MainUplinkMtu = conf.Mtu
+		//utils.WriteInfoFile()
```

Comment:
Do have the multiple instances of "//utils.WriteInfoFile()" around the code some meaning hidden meaning or it just need to be cleaned?

Reply:
forgotten 😬